pyproject.toml

```diff
@@ -16,6 +16,8 @@ dependencies = [
     'numpy>=1.23',
     'fasteners; sys_platform != "emscripten"',
     'numcodecs>=0.10.0',
+    'numcodecs[msgpack]>=0.10.0; sys_platform == "emscripten"',  # TODO: can be removed
+    'msgpack; sys_platform == "emscripten"',
```

Comment:
Both of these lines currently do not work for some reason – I am unable to import `MsgPack` from `numcodecs` even after I added them, which is why I am explicitly installing `msgpack` inside the Pyodide virtual environment for now, which seems to make this work. I will take a look at why these options do not work as I continue to work on this PR.